gr-audio/lib/alsa/alsa_sink.cc

```diff
@@ -81,7 +83,7 @@ alsa_sink::alsa_sink(int sampling_rate, const std::string device_name, bool ok_t
         error = snd_pcm_open(&t, d_device_name.c_str(), SND_PCM_STREAM_PLAYBACK, 0);
         d_pcm_handle.set(t);
         if (error < 0) {
-            boost::this_thread::sleep(boost::posix_time::milliseconds(10));
+            std::this_thread::sleep_for(std::chrono::milliseconds(10));
```

Comment:
All the cool kids do

```suggestion
            std::this_thread::sleep_for(10ms`);
```

:)

(Not a change request, just another C++ thing I kind of like)